### DISCUSSION TITLE
[Feature Request] Background Photo for Repository

### DISCUSSION THREAD

**Discussion by shukkkur** [community/community#43194] [Repositories]
### Select Topic Area

Product Feedback

### Body

<h3>Hi everyone!</h3>

<p>Not sure if I selected the right category and topic area, but here is my idea. <br>
I personally find background images visually pleasing and wish I could add a relative image to my repository. </p>

<p>For example, I have volleyball related repository and wanted an accompanying image <br>I suck at Photoshop but here, have a look at a sample</p>

![sample_design](https://user-images.githubusercontent.com/78250180/210343168-432533a8-1947-4e07-8f4a-434ed2bb8533.jpg)

<p>Not sure how to contact GitHub developers, but I hope they will see my post and think about it adding this feature</p>

**Comment by jfullstackdev** [comment]
I don't know if there are extensions that can do that

but for GitHub, by default, as far as I know, there is no option to do that 

you can only have a banner using the README file, both on your repos or your profile or your orgs

**Reply by shukkkur** [reply]
Hey @jdevstatic !) I know we can't do that yet, that's why I am proposing it and hope the GitHub developers will take that into consideration.

Perhaps you know how can I contact them, regarding this feature?

**Reply by jfullstackdev** [reply]
you're already here, where there were so many requests,

they do triaging with all of that, and if a certain feature is really that necessary, with the focus on code, then, they will surely implement it

though it's not really for Support, you can try contacting them

https://support.github.com/contact/account

**Reply by shukkkur** [reply]
Done! Thanks!
